### ISSUE TITLE
Infographic needs "Go back to homepage" button

### ISSUE THREAD

**Issue by jimaek** [jsdelivr/www.jsdelivr.com#425]
Right now if you hit the infographic page you're stuck and need to manually change the url to get our of there. Lets add some button on top and bottom to allow user to go to our homepage more easily

**Comment by MartinKolarik** [comment]
Right now it opens in a new tab.

**Comment by jimaek** [comment]
Yes but if people get there from a tweet or a different site then they are stuck
